spec/acceptance/disable_new_tasking_system_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'spec_helper_acceptance'
+
+describe 'initial configuration with newer postgresql tasking system' do
+  certdir = '/etc/pulpcore-certs'
+
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<-PUPPET
+      class { 'pulpcore':
+        worker_count          => 1,
+        use_rq_tasking_system => false,
```

Comment:
Can drop as this is the default

Reply:
I think dropping it ensures that we are properly testing the default behavior, but at the expense of making the tests less readable.

I'd rather test default behavior by adding that test to the `basic_spec.rb`

Reply:
Given that this is the subject of the test, I don't mind being explicit here.

Reply:
> Given that this is the subject of the test, I don't mind being explicit here.

This sums up my feeling as well.

Reply:
`spec/acceptance/basic_spec.rb` now tests that the PostgreSQL tasking system is used with default parameters.